sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -170,6 +170,11 @@
  *     .fromQuery("SELECT year, mean_temp FROM [samples.weather_stations]"));
  * }</pre>
  *
+ * <p>Users can optionally specify a query priority using {@link TypedRead#withQueryPriority(
+ * TypedRead.QueryPriority)} and a geographic location where the query will be executed using
+ * {@link TypedRead#withQueryLocation(String)}. Query location must be specified for jobs that are
+ * not executed in US or EU.</p>
```

Comment:
Is there a specific piece of BigQuery documentation that we can try to link to?

Reply:
Done.